Missing `conftest.py` in 3.0.3 source distribution

### Which version of Python is the problem with?

3.14, 3.13, 3.12, 3.11, 3.10

### What semver version are you using?

3.0.3

### What OS are you using? (Add more in the Environment section)

Linux

### Situation

As a side effect of removing `setuptools_scm`, the 3.0.3 source distribution has lots a lot files:

```diff
-.coveragerc
-.editorconfig
-.gitignore
-.readthedocs.yaml
 CHANGELOG.rst
-CITATION.cff
 CONTRIBUTING.rst
-CONTRIBUTORS
 LICENSE.txt
 MANIFEST.in
-Makefile
 PKG-INFO
 README.rst
-SUPPORT.md
-changelog.d/
-changelog.d/.gitignore
-changelog.d/README.rst
-changelog.d/_template.rst
-docs/
[…]
 pyproject.toml
-release-procedure.md
 setup.cfg
 src/
 src/semver/
@@ -101,9 +24,6 @@ src/semver.egg-info/dependency_links.txt
 src/semver.egg-info/entry_points.txt
 src/semver.egg-info/top_level.txt
 tests/
-tests/coerce.py
-tests/conftest.py
-tests/semverwithvprefix.py
 tests/test_bump.py
 tests/test_compare.py
 tests/test_deprecated_functions.py
@@ -119,4 +39,3 @@ tests/test_replace.py
 tests/test_semver.py
 tests/test_subclass.py
 tests/test_typeerror-274.py
-tox.ini
```

Since `tests/conftest.py` is missing, running test suite now fails due to missing fixtures, e.g.:

```pytb
_______________________________________ ERROR at setup of test_version_info_should_be_iterable ________________________________________
file /tmp/portage/dev-python/semver-3.0.3/work/semver-3.0.3/tests/test_semver.py, line 39                                              
  def test_version_info_should_be_iterable(version):                                                                                   
E       fixture 'version' not found
```

I don't know whether the other `.py` files are actually needed for tests as well.

### How to reproduce

```
wget https://files.pythonhosted.org/packages/13/f8/071c42ed3e9678f66a6d624c8df4cdd1490de2504dde6a1d47b61a8195e2/semver-3.0.3.tar.gz
tar -xf semver-3.0.3.tar.gz
cd semver-3.0.3
pip install -e . pytest
pytest
```

### Expected behavior

All tests passing.

### Environment

_No response_

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -18,6 +18,25 @@ This section covers the changes between major version 2 and version 3.
 
 .. towncrier release notes start
 
+Version 3.0.4
+=============
+
+:Released: 2025-01-24
+:Maintainer: Tom Schraitle
+
+
+Bug Fixes
+---------
+
+* :gh:`459`: Fix 3.0.3:
+
+  * :pr:`457`: Re-enable Trove license identifier
+  * :pr:`456`: Fix source dist file
+
+
+----
+
+
 Version 3.0.3
 =============
 

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,8 +1,26 @@
+include *.md
 include *.rst
 include *.txt
-include tests/test_*.py
+include CONTRIBUTORS
+include CITATION.cff
+include Makefile
+include changelog.d/*
+graft docs/**
+include tests/*.py
+include tox.ini
+include .pytest.ini
+include .ruff.toml
+include uv.lock
+
+# The dot files:
+include .coveragerc
+include .editorconfig
+include .gitignore
+include .readthedocs.yaml
+
 
 prune docs/_build
 recursive-exclude .github  *
+prune docs/**/__pycache__
 
 global-exclude __pycache__

--- a/docs/usage/semver-version.rst
+++ b/docs/usage/semver-version.rst
@@ -4,4 +4,4 @@ Getting the Version of semver
 To know the version of semver itself, use the following construct::
 
    >>> semver.__version__
-   '3.0.3'
+   '3.0.4'

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -35,7 +35,7 @@ classifiers = [
     "Environment :: Web Environment",
     "Intended Audience :: Developers",
     "Development Status :: 5 - Production/Stable",
-    # "License :: OSI Approved :: BSD License",
+    "License :: OSI Approved :: BSD License",
     "Operating System :: OS Independent",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3.7",

--- a/src/semver/__about__.py
+++ b/src/semver/__about__.py
@@ -16,7 +16,7 @@ of the semver specifictation, author, maintainers, and description.
 """
 
 #: Semver version
-__version__ = "3.0.3"
+__version__ = "3.0.4"
 
 #: Original semver author
 __author__ = "Kostiantyn Rybnikov"

